src/vs/workbench/parts/files/browser/files.contribution.ts

```diff
@@ -267,7 +267,7 @@ configurationRegistry.registerConfiguration({
 		'files.hotExit': {
 			'type': 'string',
 			'enum': [HotExitConfiguration.OFF, HotExitConfiguration.ON_EXIT, HotExitConfiguration.ON_EXIT_AND_WINDOW_CLOSE],
-			'default': HotExitConfiguration.ON_EXIT,
+			'default': HotExitConfiguration.OFF,
```

Comment:
I just merged your change to force off HotExit.  you probably should merge that change and set it to read from the config and default to OFF.